tests/Hamcrest/Type/IsNumericTest.php

```diff
@@ -25,7 +25,10 @@ public function testEvaluatesToTrueIfArgumentMatchesType()
         assertThat('0.053e-2', numericValue());
         assertThat('-53.253e+25', numericValue());
         assertThat('+53.253e+25', numericValue());
+        assertThat(0x4F2a04, numericValue());
         assertThat('0x4F2a04', numericValue());
+        assertThat('0x', numericValue()); // I think that it's valid hexadecimal but I am not 100% sure.
```

Comment:
Doesn't look like valid hexadecimal. I guess you'll get syntax error if you type `$a = 0x;`.